pages/linux/blight.md

```diff
@@ -0,0 +1,24 @@
+# blight
+
+> Utility for changing the display brightness.
+> More information: <https://github.com/gutjuri/blight>.
+
+- Set display brightness to 50%:
+
+`blight set {{50}} -r`
+
+- Show current display brightness:
+
+`blight show`
+
+- Show maximum display brightness:
```

Comment:
This description is not clear to me, prints the value or applies the max brightness to the screen?

Reply:
It prints the value.

Reply:
Changed the command to "Print maximum display value" in [blight: fix arguments and fix command desc](https://github.com/tldr-pages/tldr/pull/8633/commits/5149693ccbee60351349bd00501c0f4d5f6d07d1)